projects/angular/src/data/datagrid/_datagrid.clarity.scss

```diff
@@ -656,7 +656,6 @@
     flex-direction: column;
     flex: 1 1 auto;
     align-content: flex-start;
-    position: relative;
```

Comment:
This line was supposed to be needed by a corner case scenario that no-one from the old team seemed to remember. I tested this for more than 10 days, while working on another datagrid issue and observed no visible regression. The fix does not just remove the `position: absolute` from here. It actually relocates it to the `datagrid-host` parent, so popovers only get reanchored, still inside the datagrid.